apps/hash-frontend/src/blocks/block-collection/block-view.tsx

```diff
@@ -280,10 +281,12 @@ export class BlockView implements NodeView {
                 }}
                 onClick={this.onDragEnd}
               />
-              <CreateBlockCommentButton
-                blockEntityId={blockEntityId}
-                rootNode={this.rootNode}
-              />
+              {this.isCommentingEnabled ? (
+                <CreateBlockCommentButton
+                  blockEntityId={blockEntityId}
+                  rootNode={this.rootNode}
+                />
```

Comment:
This looks like it will enable commenting even if the page is in `readonly` mode... we should preserve the existing behaviour, which is to not show the commenting interface if the page is in `readonly`

Reply:
Thanks for catching, have addressed in https://github.com/hashintel/hash/pull/3367/commits/0772cc63551a4a9da4b55de6fae67a526aa40062.